src/pages/ProblemPage.tsx

```diff
@@ -25,17 +25,62 @@ const items: TabsProps['items'] = [
         key: R_CC,
         label: 'CodeChef',
         children: (
-            <ProblemFilter radios={[
-                { label: 'All', value: '' },
-                { label: 'Rated for All', value: 'All' },
-                { label: 'Rated till 6-stars', value: '6.[Ss]tars' },
-                { label: 'Rated till 5-stars', value: '5.[Ss]tars' },
-                { label: 'Rated for Divs 2, 3 & 4', value: 'Divs 2' },
-            ]}></ProblemFilter>        
+            <div style={{ position: 'relative', padding: '20px' }}>
+                <ProblemFilter radios={[
+                    { label: 'All', value: '' },
+                    { label: 'Rated for All', value: 'All' },
+                    { label: 'Rated till 6-stars', value: '6.[Ss]tars' },
+                    { label: 'Rated till 5-stars', value: '5.[Ss]tars' },
+                    { label: 'Rated for Divs 2, 3 & 4', value: 'Divs 2' },
+                ]}></ProblemFilter>
+                <div style={{
+                        position: 'absolute',
+                        top: '-12px',
+                        right: '15px',
+                        padding: '8px',
+                        borderRadius: '4px', 
+                        width: '300px',
+                        display : "flex",
+                        flexWrap : "wrap",
+                        overflowX: 'auto', // Adds horizontal scroll if needed
+                        //whiteSpace: 'nowrap', // Ensures content stays in a single line if necessary
+                        //backgroundColor : 'red', 
+                }}>
+                    <table style={{ width: '100%', borderCollapse: 'collapse' ,tableLayout: 'fixed'}}>
+                        <tbody>
+                            <tr>
+                                <td style={{ padding: '0 5px', textAlign: 'center' }}>
+                                    <div style={{ display: 'flex', alignItems: 'center' }}>
+                                        <span style={{ width: '20px', height: '5px', backgroundColor: 'red', marginRight: '5px' }}></span>
+                                        <span >Div 1</span>
+                                    </div>
+                                </td>
```

Comment:
How about using a loop like ProblemList.jsx? And use class names to control different styles by SCSS variables.